.travis.yml

```diff
@@ -2,6 +2,9 @@ language: python
 
 before_script:
 - pip install tox
+- sed -i 's/\/opt/\/home\/travis\/build\/rcbops/g' ansible-role-requirements.yml
+- pip2 install --force-reinstall 'ansible===1.9.4'
+- ansible-galaxy install --role-file=ansible-role-requirements.yml --ignore-errors --force
```

Comment:
this was copied from upstream openstack-ansible `scripts/bootstrap-ansible.sh` looks like it was done in the megapatch.  https://github.com/openstack/openstack-ansible/commit/8e6dbd01c9924ebecffdc04f74e4f6d042610ed8